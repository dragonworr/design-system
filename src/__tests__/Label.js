import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import { Label } from '..'

describe('Label', () => {
  test('it renders', () => {
    const json = renderer.create(<Label />).toJSON()
    expect(json).toMatchSnapshot()
  })
})
