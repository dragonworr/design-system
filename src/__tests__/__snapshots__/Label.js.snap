// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label it renders 1`] = `
.c0 {
  font-size: 10px;
  display: block;
  width: 100%;
  font-size: 10px;
}

<label
  className="c0"
  fontSize="10px"
/>
`;
